cel/io.go

```diff
@@ -25,6 +25,14 @@ import (
 
 // CheckedExprToAst converts a checked expression proto message to an Ast.
 func CheckedExprToAst(checkedExpr *exprpb.CheckedExpr) *Ast {
+	return CheckedExprToAstWithSource(checkedExpr, nil)
+}
+
+// CheckedExprToAstWithSource converts a checked expression proto message to an Ast,
+// using the provided Source as the textual contents.
+//
+// Use CheckedExprToAst if you do not have a source.
```

Comment:
Thanks for waiting. I had to think a bit about how this should be phrased. I would prefer the comment read as follows:

```
CheckedExprToAstWithSource converts a checked expression proto message to an Ast,
using the provided Source as the textual contents. 

In general the source is not necessary unless the AST has been modified between the
`Parse` and `Check` calls as an `Ast` created from the `Parse` step will carry the source
through future calls.

Prefer CheckedExprToAst if loading expressions from storage.
```